trackintel/preprocessing/positionfixes.py

```diff
@@ -166,10 +167,7 @@ def generate_staypoints(
 
 
 def generate_triplegs(
-    positionfixes,
-    staypoints=None,
-    method="between_staypoints",
-    gap_threshold=15,
+    positionfixes, staypoints=None, method="between_staypoints", gap_threshold=15, print_progress=False
```

Comment:
Please add a comma at the end of this statement such that the formatting does not change with black. Please check the original version.